nntrainer/compiler/ini_interpreter.cpp

```diff
@@ -66,8 +66,9 @@ class BackboneLayer {};
  * @param sec_name section name
  * @return std::vector<std::string> list of properties
  */
-std::vector<std::string> section2properties(dictionary *ini,
-                                            const std::string &sec_name) {
+std::vector<std::string> section2properties(
+  dictionary *ini, const std::string &sec_name,
+  std::function<const std::string(std::string)> &pathResolver) {
```

Comment:
add @param about pathResolver?

Reply:
If relative path is given to ini, it is priortized to be interpreted relative to ini file. This resolver is used to interprete relative path.

Added @param for detail :)